Makefile

```diff
@@ -55,12 +55,21 @@ install-travis:
 # Example:
 #   make check
 #   make test
-#   make check WHAT=pkg/build TESTFLAGS=-v
-check:
+#   make check WHAT=pkg/docker TESTFLAGS=-v
+check:	verify
```

Comment:
but what is the difference between calling $(MAKE) test and having test as a dependency of the check target?